### PR TITLE
Infinite scroll

### PR DESCRIPTION
**Pull request by alejandrobabio**
Related task: https://www.pivotaltracker.com/story/show/91684570

### CODE REVIEW

File: app/controllers/event_instances_controller.rb [hunk 1]
```diff
@@ -16,8 +16,9 @@ def update
   end
 
   def index
-    @event_instances = (params[:live] == 'true') ? EventInstance.live : EventInstance.latest
-    render partial: 'hangouts' if request.xhr?
```
Comment (alejandrobabio) [inline app/controllers/event_instances_controller.rb]: I remove line 20, because on js request, I want to respond with ```views/event_instances/index.js.erb```. I think it was dead code.

File: features/hangout.feature [hunk 2]
```diff
@@ -158,3 +158,21 @@ Feature: Managing hangouts of scrums and PairProgramming sessions
...
+    Then I should see 30 hangouts
+    And I should see link "Next"
+    And I should not see "No more hangouts"
+
+  @javascript
+  Scenario: Infinite scroll on hangouts scroll down 2 times
+    Given 70 hangouts exists
+    When I visit "/hangouts"
+    And I scroll to bottom of page
+    And I scroll to bottom of page
```
Comment (tochman) [inline features/hangout.feature]: Why is this line here twice?

**Comment by alejandrobabio** [comment]
Because I want to scroll 2 times, and show 70 hangouts. I want to be sure that after first get script, javascript keeps working.

File: features/hangout.feature (hunk 2 above)
Comment (tochman) [inline features/hangout.feature]: I just wonder if we should check for what we see after the first scroll.

Comment (alejandrobabio) [inline features/hangout.feature]: Ok. And I will refactor this one, because it is no clear at first read.

Comment (tochman) [inline features/hangout.feature]: Great, thank you.

**Comment by alejandrobabio** [comment]
@tochman I think it's ready.

**Comment by alejandrobabio** [comment]
I close this one, because I'll send a new PR with all the changes.
